docs/extending/kiotaabstractions.md

```diff
@@ -6,82 +6,107 @@ parent: Understand and Extend the Kiota generator
+
+    Task SendNoContentAsync(RequestInformation requestInfo,
+                            IResponseHandler responseHandler = default);
+}
 ```
 
-Kiota service libraries return the model type that is associated with HTTP resource. This behavior can be overriden by changing the `responseHandler` to do something different than default behavior.  One use of this is to change the response type to be either a native HTTP response class, or return a generic API response class that provides access to more underlying metadata.
+Kiota service libraries return the model type that is associated with HTTP resource. This behavior can be overridden by changing the `responseHandler` to do something different than default behavior.  One use of this is to change the response type to be either a native HTTP response class, or return a generic API response class that provides access to more underlying metadata.
 
-## RequestInformation
+### Request Information
 
 In order to enable Kiota service libraries to make requests, they need to be able accumulate information about the request and pass it to the core library. The RequestInformation object is designed to do that. It only contains properties that be provided by the request builders. As request builders get more sophisticated, so may the RequestInformation class.
 
 ```csharp
 public class RequestInformation
-    {
-        public Uri URI { get; set; }
-        public HttpMethod HttpMethod { get; set; }
-        public IDictionary<string, object> QueryParameters { get; set; } = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
-        public IDictionary<string, string> Headers { get; set; } = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
-        public Stream Content { get; set; }
-    }
+{
+    public string UrlTemplate { get; set; }
+    public Uri URI { get; set; }
+    public HttpMethod HttpMethod { get; set; }
+    public IDictionary<string, object> QueryParameters { get; set; } = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
+    public IDictionary<string, string> Headers { get; set; } = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+    public IDictionary<string, object> PathParameters { get; set; } = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
+    public Stream Content { get; set; }
+    public IEnumerable<IRequestOption> RequestOptions { get }
+}
 ```
 
-```TypeScript
-export interface RequestInformation {
-    URI?: URL;
-    httpMethod?: HttpMethod;
-    content?: ReadableStream;
-    queryParameters?: Map<string, object>;
-    headers?: Map<string, string>;
+### Response Handler
+
+When passed to the execution method from the fluent style API, this allows core to do all the default hard work, but enables a custom response handler to change the behavior of and access the native response object.
+
+```CSharp
```

Comment:
Always use `csharp` here. Jekkyl fails to render properly with capitalized language names.

Reply:
```suggestion
```csharp
```

Reply:
fixed!